org.eclipse.languageserver/src/org/eclipse/languageserver/LSPStreamConnectionProviderRegistry.java

```diff
@@ -102,15 +104,15 @@ private void persist() {
 			}
 		}
 	}
-	
+
 	public List<StreamConnectionProvider> findProviderFor(final IContentType contentType) {
-		return Arrays.asList(connections
-			.stream()
-			.filter(entry -> { return entry.getKey().equals(contentType); })
-			.map(entry -> { return new LaunchConfigurationStreamProvider(entry.getValue()); })
-			.toArray(StreamConnectionProvider[]::new));
+		return Arrays.asList(connections.stream().filter(entry -> {
```

Comment:
the formatting here is much more complicated to read than earlier. It's a regular issue with automatic formatting when doing some "fluent APIs",